com.github.vikdevelop.photopea_app.yml

```
@@ -0,0 +1,26 @@
+app-id: com.github.vikdevelop.photopea_app
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: photopea
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --filesystem=xdg-pictures
+modules:
+  - name: photopea
+    buildsystem: simple
+    build-commands:
+    - install -Dm755 -t /app/bin photopea
+    - install -Dm755 -t /app/share/applications com.github.vikdevelop.photopea_app.desktop
+    - install -Dm644 -t /app/share/icons/hicolor/128x128/apps/com.github.vikdevelop.photopea_app.png data/icons/com.github.vikdevelop.photopea_app.png  
```

```suggestion
    - install -Dm644 -t /app/share/icons/hicolor/128x128/apps/ data/icons/com.github.vikdevelop.photopea_app.png  
```

`-t` specifies a target directory